FlorenceBML/BMLacq810/BMLacq810.xml

```diff
@@ -35,8 +35,447 @@
+                <incipit xml:lang="gez">ጸሎተ፡ ሣልስ፡ ዘይትነበብ፡ ላዕለ፡ ካህናት፡ በከመ፡ ዓብየት፡ ምግባሪነ፡ እግዚኦ፡ ወኵሎ፡ በጥበብ፡ ገበርከ፡ መልአ፡ ምድረ፡ እምፍጥረትከ። ዘአንተ፡ ሕይወትኒ፡ ወሠራዕከ፡ ሞትኒ። ወናሁ፡ የሐይዉ፡ በኀቤከ፡ ኵሎሙ፡ 
+                  እለ፡ ይደልዎሙ፡ ሰበይከ፤ እስመ፡ አንተ፡ አምላከነ፡ ሕይው፡ ያው፡ ዘይነብር፡ ለዓለም፡ ለብፁዓዊተ፡ ዘእንበለ፡ ተፍጻሜት፡ በእንተ፡ አእምሮ፡ ብከ። </incipit>
+                <explicit xml:lang="gez">አዕርጎ፡ ኀበ፡ ገነተ፡ ትፍሥሕት፡ አብልኦ፡ እምዕፀ፡ ሕይወት። አርፍቆ፡ ውስተ፡ ሕጽነ፡ አበዊነ፡ አብርሃም፡ ይስሐቅ፡ ወያዕቆብ፡ ውስተ፡ መንግሥትከ። ወበከመ፡ ንሕነ፡ ነሐሥሥ፡ ምሕረተ፡ ውስተዝ፡ መካን፡ ውእቱኒ፡ 
+                  ካዕበ፡ ይዘከረነ፡ በቅድሜከ፡ በጸጋ፡ እስከ፡ ለዓለመ፡ ዓለም፡ አሜን። </explicit>
+              </msItem>
+              <msItem xml:id="ms_i1.44">
+                <locus from="61vb" to="64ra"/>
+                <title ref="LIT0000">Fifth prayer for the apostle Matthew</title>
+                <incipit xml:lang="gez">ዝኬ፡ ሰቆቃው፡ ኃምስ፡ ጸሎት፡ ዘማቴዎስ፡ ሐዋርያ፡ ትብል፡ ዘከመዝ። እባርከከ፡ እግዚእየ፡ በኵሉ፡ ጊዜ፡ ዘኢይበሊ፡ መንበሩ፡ ልዑል፡ ላዕለ፡ ኵሉ፡ መልዕልት፡ እሴብሐከ፡ ዘኢኃብአ፡ ርእሶ፡ ዳእሙ፡ አግብኦ፡ በእንተ፡ 
+                  ኃጣውኢነ፡ ከመ፡ ያውጽአነ፡ እም<gap reason="illegible"/>ሕተት። </incipit>
+                <explicit xml:lang="gez">የለሶን፡ ክርስቶስ፡ ዘበአማን፡ አምላክነ፡ በሰዕለታ፡ ለፍሥሕት፡ እምከ፡ ረሰዮ፡ ለነፍስ፡ ገብርከ፡ እገሌ፡ ውስተ፡ ሕጽነ፡ አብርሐም፡ ይስሐቀ፡ ወያዕቆብ፡ ደምር፡ 
+                  ነፍስ፡ ወኪያነሂ፡ ተሣሃለነ፡ በኵሉ፡ ጊዜ፡ ወበኵሉ፡ ሠዓት። ይእዜኒ፡ ወዘልፈኒ፡ ወለዓለመ፡ ዓለም፡ አሜን፤ ወአሜን፤ ለይኩን። </explicit>
+              </msItem>
+              <msItem xml:id="ms_i1.45">
+                <locus from="64ra" to="66rb"/>
+                <title ref="LIT0000">Prayer for repentance</title>
+                <incipit xml:lang="gez"> ጸሎተ፡ ንሥሐ፡ ዓቢይ፡ ጸሊ፡ ዘንተ፡ በተፍጻሜተ፡ ኵሉ፡ እግዚእ፡ እግዚአብሔር፡ አኃዜ፡ ኵሉ፡ ዓለም፤ ብሑተ፡ ሥልጣን፡ ወስም፡ ቀዳማዊ።
+                  ዘእንበለ፡ ጥንት፡ ወደኅራዊ፡ ዘእንበለ፡ ተፍጻሜት፡ ዘትነብር፡ መልዕልተ፡ አርያም፡ ዘልብሰ<gap reason="illegible"/> ወመንጠለዕትከ፡ <gap reason="illegible"/></incipit>
+                <explicit xml:lang="gez">አስሉሥ <gap reason="lost"/>ብ፡ ወብ፡ ወወልድ፡ ወመንፈስ፡ ኢስከ፡ ነፍ፡ ደኃሪት። ወላዕሌነ፡ ይኩን፡ ሣህልከ፡ ወምሕረትከ፡ ይእዜኒ፡ 
+                  ወዘልፈኒ፡ ወለዓለም፡ አሜን። </explicit>
+              </msItem>
+              <msItem xml:id="ms_i1.46">
+                <locus from="66rb" to="68va"/>
+                <title ref="LIT0000">Funeral ritual for fathers and young men</title>
+                <incipit xml:lang="gez">አርእስተ፡ ምንባብ፡ ላዕለምዉታን፡ ሶበ፡ ይመውቱ፡ አበው፡ ወራዙት። ትብል፡ ጽሎተ፡ አኮቴት፡ ወጸሎተ፡ ዕጣን። ወእምዝ፡ ትብል፡ ጸሎተ፡ ቀዳሕ፡ 
+                  ዘእምዳዊት፡ በዜማ፡ ግንፅዝ፤ ዘ፻፴፰፡ አይቴኑ፡ አኃውር፡ እመንፈስከ፡ ወአይቴኑ፡ እጐይይ፡ እምቅድመ፡ ገጽከ፡ እመ<gap reason="illegible"/>፡ ውስተ፡ ሰማይ፡ 
+                  <gap reason="illegible"/>።</incipit>
+                <explicit xml:lang="gez">ለከ፡ ሰብሐት፡ ወለአቡከ፡ ኄር፡ ምንፈስ፡ ቅዱስ፡ እስከ፡ ለዓለም፡ ዓለም፡ ዘይመጽእ፡ ወእምድኅረ፡ አቡነ፡ ዘበሰማያት፡ ወይበል፡ ካህን፡ ጸሎተ፡ ንስሐ፡ 
+                  ዘወልድ፡ ወይበሉ፡ እግዚኦ፡ መሐሪነ፡ ፵ወ፩፡ እስከ፡ ይከውን። ወእምድኅረ፡ ቀበርዎ፡ ይበል፡ ካህን፡ ጸሎተ፡ መቃብር፡ ወፍተሐተ፡ መቃብር፡ ዳግመ፤ ወካዕበ፡ ይብል፡ በከመ፡ ይቤ፡ ሰሎሞን።</explicit>
+                <note>According to <bibl><ptr target="bm:Marrassini1986Laurenziana"/><citedRange unit="page">115</citedRange></bibl> 
+                  similar to ANL, 71:16, BN(Gr.) 337:XIV(?) and JRyl 7:15.</note>
```

Comment:
either link to stub records or at least expand shelfmarks to make them understandable

Reply:
For the manuscript in Vienna I checked the catalogues of Müller and Rhodokanakis. I found a manuscript Aeth. 18, that is a Maṣḥafa Gənzat and no. 9 in Rhodokanakis. It is apparently the account, that Marrassini refers to as Wien 9. But textpart no. 16 is very unspecific and does not give any conclusions to the content more than guessing. I will nevertheless encode it in the way that Rhodokanakis has recorded and Marrassini refers to as "ONBAeth18#ms_i1.16". Is that correct?

Reply:
For the Rylands library in Manchester I don't have access to the catalogue of Strelcyn (bm:Strelcyn1974CatJRylands). Is it neccessary to create a stub in order to know the correct idno?

Reply:
The same is for the collection Griaule and the catalogue by again Strelcyn: bm:Strelcyn1954Griaule

Reply:
Both Strelcyn catalogues are available in our common electronic library in PDF which you have access to.
Outside of UHH you will need to dial in through VPN and then you will have access to the folder (I remember I installed it so it should be visible in your explorer) `Z:\gw\aai-hlz-research\ELEKTROBIBLIOTHEK` or alternatively from anywhere simply through your browser (but slower loading times of course) https://uhhshare.uni-hamburg.de/ -> Netzwerkordner ->gw ->aai-hlz-research -> ELEKTROBIBLIOTHEK

Reply:
Thank you! I found them.

Reply:
I made a record for JRyl7.